config/v1/types_network.go

```diff
@@ -90,10 +90,12 @@ type NetworkSpec struct {
 // NetworkStatus is the current network configuration.
 type NetworkStatus struct {
 	// IP address pool to use for pod IPs.
+	// +listType=atomic
```

Comment:
Are the fields in the ClusterNetworkEntry unique? Eg can you only have one entry for each CIDR value? Wondering if this should actually be a list type with a map key 🤔

Reply:
The values in both arrays are unique.
They are a copy of what is being provided in the spec after it gets processed and validated by CNO.
Is there something to gain from using a map here?

Reply:
For a little more context in most cases there is just one entry in each array (two in dualstack). 
I also think that if we use a map listType this could potentially cause with multiple owners managing different values. This is just a theoretical scenario for now since CNO is the only component managing network.config status.

Reply:
Ack, lets stick with atomic in this case.